mmaction/apis/train.py

```diff
@@ -1,22 +1,25 @@
 import copy as cp
+import os.path as osp
 
 import torch
 from mmcv.parallel import MMDataParallel, MMDistributedDataParallel
 from mmcv.runner import (DistSamplerSeedHook, EpochBasedRunner, OptimizerHook,
-                         build_optimizer)
+                         build_optimizer, get_dist_info)
 from mmcv.runner.hooks import Fp16OptimizerHook
 
 from ..core import (DistEvalHook, EvalHook, OmniSourceDistSamplerSeedHook,
                     OmniSourceRunner)
 from ..datasets import build_dataloader, build_dataset
 from ..utils import PreciseBNHook, get_root_logger
+from .test import multi_gpu_test
 
 
 def train_model(model,
                 dataset,
                 cfg,
                 distributed=False,
                 validate=False,
+                test=False,
```

Comment:
rename it with `test_model_after_train` may be better.

Reply:
I think using the name `test` is of the same style with `validate`.

Reply:
default value for `test` should be `dict`